versions/ufo3/glyphs/glif.md

```diff
@@ -167,6 +167,18 @@ This element may occur any number of times.
 
 #### This element has no child elements.
 
+#### Anchor naming conventions
+
+The following are agreed upon standard naming conventions for the `name` of anchors for special uses.
+
+##### Ligature Carets
+
+Anchors with a `name` starting with `caret_` or `vcaret_` indicate horizontal or vertical caret positions for ligature glyphs, for example to compile the [OpenType GDEF Ligature Caret List table]. Anchors with the name `caret_` provide the `x` value for a ligature caret, and the `y` value is ignored. Likewise, anchors with the name `vcaret_` provide the `y` values for a ligature caret, and the `x` value is ignored.
```

Comment:
Sorry, a few more nitpicky things:
"Anchors with the name `caret_`" -> "Anchors with a name starting with `caret_`"
"value for a ligature caret" -> "value for a ligature caret position"
"anchors with the name `vcaret_`" -> "anchors with a name starting with `vcaret_`"
"provide the `y` values for a ligature caret" -> "provide the `y` value for a ligature caret position"